### PR TITLE
solved the ui issues in workspace setting and members section #3654

### PR DESCRIPTION
**Pull request by parthosc**
Close: #3654 

## PR Details

fixed the styles of workspace member/setting according to figma file attached in issue

## PR Checklist

- [ ] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

image

### CODE REVIEW

**Comment by parthosc** [comment]
image

**Comment by parthosc** [comment]
image

**Comment by GreenMachine01** [comment]
@alexbass86 please review the screenshots

**Comment by GreenMachine01** [comment]
@souravjain540 see feedback from Alex above.

**Comment by parthosc** [comment]
@alexbass86 let me check figma design again and set exact size of the icon and other ui

File: packages/amplication-client/src/Workspaces/MemberListItem.scss [hunk 1]
```diff
@@ -18,19 +18,18 @@
...
-    justify-content: center;
-    padding: 0 var(--default-spacing);
-    background-color: rgba(49, 197, 135, 0.09);
+    font-family: Poppins;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    letter-spacing: 0em;
+    text-align: left;
+    color: #ffbd70;
```
Comment (abrl91) [inline packages/amplication-client/src/Workspaces/MemberListItem.scss:31]:
```suggestion
  @include regular-12
```

I'm unsure about the color, but if what we have in regular-12, is not what you need, you can override it with the color property, just use vars, for example `color: var(--black80);`

**Review by abrl91 — CHANGES_REQUESTED** [review]

File: packages/amplication-client/src/Workspaces/MemberListItem.scss [hunk 2]
```diff
@@ -18,19 +18,18 @@
   }
   &__description {
     @include body2;
-    color: var(--black80);
+    color: #53dbee;
```
Comment (morhag90) [inline packages/amplication-client/src/Workspaces/MemberListItem.scss:21]: Please use a var instead:
color: var(--black80); (just an example).

**Comment by souravjain540** [comment]
@parthosc  I'd like to offer you some swag for this PR, please make the changes and DM me on Discord :)

**Comment by parthosc** [comment]
@alexbass86 @souravjain540 

For this [review](https://github.com/amplication/amplication/pull/5570#issuecomment-1512650225), can you confirm that the padding of the `workspace member` ui is different for owner and invited members, I've to make both similar according to figma attached in the issue, right?

**Comment by alexbass86** [comment]
> 

yes sir!

**Comment by parthosc** [comment]
@alexbass86 @souravjain540 
image

**Comment by alexbass86** [comment]
Approved, great job @parthosc. Thank you for the contribution :)

**Review by abrl91 — APPROVED** [review]

**Pull request merged**